README.md

```diff
@@ -12,6 +12,20 @@ Please note that this is still an early **work in progress**, and everything in
 
 - [Criteria for the `severity` and `user_base_impact` fields](./docs/severity-and-impact.md).
 
+## Script to create yml file for a given bugzilla bug
```

Comment:
Can we maybe move this into the `docs/` folder and link to it in README.md, like we did with the severity docs above? I fully assume there will be more documentation over time, and having it split into separate files makes documentation a bit easier to discover and maintain :)

Reply:
Sounds good!